Raven.Voron/Voron/Trees/Page.cs

```diff
@@ -172,6 +172,19 @@ public ushort PageSize
             }
         }
 
+        public List<long> GetAllOverflowPages()
+        {
+            var results = new List<long>(NumberOfEntries);
```

Comment:
I most certainly do know whatever they are overflow or not.
See: 
https://github.com/ayende/ravendb/blob/master/Raven.Voron/Voron/Trees/NodeHeader.cs#L21

You should only have this for items that have overflow set.
Otherwise, you are trying to prefetch an invalid memory location, and I don't know what the result of that is, but it isn't likely to be good.